doc/guides/security-release-process.md

```diff
@@ -2,7 +2,7 @@
 
 The security release process covers the steps required to plan/implement a
 security release. This document is copied into the description of the Next
-Security Release, and used to track progess on the release. It contains ***TEXT
+Security Release, and used to track process on the release. It contains ***TEXT
```

Comment:
This is a typo, but I think it's meant to say "progress" not "process."

Reply:
It is great to see you review my code, I've revised the text and committed.